rcl/test/rcl/test_subscription.cpp

```diff
@@ -152,12 +150,34 @@ TEST_F(CLASSNAME(TestSubscriptionFixture, RMW_IMPLEMENTATION), test_subscription
   subscription = rcl_get_zero_initialized_subscription();
   EXPECT_FALSE(rcl_subscription_is_valid(&subscription));
   rcl_reset_error();
+}
 
-  // Check that valid subscriber is valid
-  subscription = rcl_get_zero_initialized_subscription();
+/* Basic nominal test of a subscription: valid subscriber is valid.
```

Comment:
```suggestion
/* Basic nominal test of a subscription.
```

And I would change the description of the previous test to "Test subscription init, fini and is_valid functions".

Reply:
Thanks, done by 3173e8790dc361a4b1ccace8e7710ebf7a863d64.
To change both , I didn't use "Commit suggestion".